.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -0,0 +1,295 @@
+              description: The kubeconfig of new cluster will be stored onto the volume backing this Workspace
+          kind: ClusterTask
+          steps:
+            - name: generate-hypershift-deployment
+              image: ghcr.io/sharkusmanch/envsubst:1.2.0
+              env:
+                - name: REGION
+                  value: "$(params.region)"
+                - name: CLUSTER_NAME
+                  value: "$(params.cluster-name)"
+                - name: IMAGE
+                  value: "$(params.image)"
+              script: |
+                envsubst < $(workspaces.hypershift-cluster-template.path)/cluster-template.yaml | tee $(workspaces.output.path)/HypershiftDeployment.yaml
+            - name: deploy-cluster
+              image: quay.io/xinjiang/kubectl:1.25
+              script: |
+                #!/usr/bin/env bash
+                set -o errexit
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+                kubectl -n $(params.namespace) apply -f $(workspaces.output.path)/HypershiftDeployment.yaml
+                echo "Wait until hypershift cluster is ready..."
+                wait_period=0
+                while [[ "$(kubectl -n $(params.namespace) get hypershiftdeployment $(params.cluster-name) -o json | jq -r '.status.conditions[]? | select(.type == "HostedClusterProgress") | .reason')" != "Completed" ]]
+                do
+                  if [ "$wait_period" -gt 2500 ]; then
+                    echo "Failed to create OCP cluster."
+                    kubectl -n $(params.namespace) get hypershiftdeployment $(params.cluster-name) -o yaml
+                    exit 1
+                  fi
+                  sleep 60
+                  wait_period=$((wait_period + 60))
+                  echo "Waited $wait_period seconds..."
+                done
+
+                echo "Hypershit is ready, the following is the cluser kubeconfig"
+                kubectl get secret -n clusters "$(params.cluster-name)-admin-kubeconfig" -o json | jq -r '.data.kubeconfig'  | base64 -d | tee $(workspaces.output.path)/kubeconfig
+      - name: fetch-repository
+        runAfter:
+          - "deploy-cluster"
+        taskRef:
+          name: git-clone
+          kind: ClusterTask
+        workspaces:
+          - name: output
+            workspace: source
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+      - name: ckcp-test
+        runAfter:
+          - "fetch-repository"
+        workspaces:
+          - name: source
+            workspace: source
+          - name: kubeconfig-dir
+            workspace: shared-workspace
+        taskSpec:
+          workspaces:
+            - name: source
+            - name: kubeconfig-dir
+          kind: ClusterTask
+          steps:
+            - name: create-podman-container
+              image: quay.io/xinjiang/kubectl:1.25
```

Comment:
Change this to the official image. If this is a custom image you've built, then please upload this to quay.io/redhat-pipeline-service organization.

Reply:
fixed, same as above